jquery.kinetic.js

```diff
@@ -226,6 +224,13 @@
         if (self._useTarget(e.target, e) && self.elementFocused){
           return false;
         }
+      },
+      // prevent selection when dragging
+      selectStart: function (e){
+        if ($.isFunction(self.settings.selectStart)){
```

Comment:
>just allow selection when `!self._useTarget(e.target, e)`

that's what I did in the previous commit (tsaikd/jquery.kinetic@f0bc356)
it returns false if `self._useTarget(e.target, e)`